tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -943,7 +971,59 @@ def test_pf_store_non_scalar_plugin_data_without_shape_issue_171():
         events = h5["events"]
         assert "image_gauss_filter" in events.keys()
         feat = h5["events"]['image_gauss_filter']
+        fshape = feat.shape
+        assert len(fshape) == 3 and fshape[1] > 3 and fshape[2] > 3
```

Comment:
You can have these assertions on separate lines for better clarity. Also, what shape is dim[0]?

Reply:
Same comment for the other tests